### PR TITLE
[Post Shader] FXAA with natural color

### PR DESCRIPTION
**Pull request by weihuoya**
FXAA
screenshot_20190202-215529_dolphin

FXAA with natural color, a little color enhance. natural shader port from Libretro, 
screenshot_20190202-215448_dolphin

### CODE REVIEW

**Comment by weihuoya** [comment]
There are many useless post shaders in dolphin, should we remove these files?

**Comment by RinMaru** [comment]
aside from FXAA I don't see any use in any of the PP shaders... oh and Stereoscopic. this one looks nice

**Comment by kevinvanrijn** [comment]
The [first shaders](https://github.com/dolphin-emu/dolphin/commits/36026c62edb903ed935da8df1d0541727685a8ef/Data/User/Shaders) were added to test the feature. Then [more](https://github.com/dolphin-emu/dolphin/tree/95b39da7cad68e1bd1a9c8be713ab1df7988f3d0/Data/User/Shaders) and [more](https://github.com/dolphin-emu/dolphin/commits/f8c01efece74d1eaec91f560d6f13c6782bbaba8) got added just to show off some neat graphical effects. https://github.com/dolphin-emu/dolphin/pull/348#issuecomment-42326069

I can only assume they've stuck around this long because they've remained useful to test with while keeping the feature updated and because they haven't gotten in the way of anything yet.

They're not really useful for users though.

---

This might as well get added if those "dumb shaders" are still the bar. It's just a shame that it's not possible to separate this pull's color enhance and FXAA into two shader files and have them both enabled.

**Comment by weihuoya** [comment]
it's convenient for android phone apply this shader, but unnecessary for PC.

File: Data/Sys/Shaders/FXAA_natural.glsl [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                      1, -1.10698902,  1.70461500);
+
+const float3 YIQ_lo = float3(0, -0.595716, -0.522591);
+const float3 YIQ_hi = float3(1,  0.595716,  0.522591);
+
+float4 applyNatural(float3 c)
+{
+    c = pow(c, float3(GIN, GIN, GIN));
+    c = MUL(RGBtoYIQ, c);
+    c = float3(pow(c.x, Y), c.y * I, c.z * Q);
```
Comment (Tilka) [inline Data/Sys/Shaders/FXAA_natural.glsl]: Is this just increasing brightness and saturation? I'm a bit skeptical about this function.

**Pull request closed without merging**